### PR TITLE
Adding marketimage related upgrade jobs

### PR DESCRIPTION
**Pull request by JianLi-RH**

### CODE REVIEW

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.10-amd64-nightly-4.10-upgrade-from-stable-4.9-azure-ipi-private-workers-marketimage-p2-f14

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly-4.10-upgrade-from-stable-4.9.yaml [hunk 1]
```diff
@@ -337,6 +337,18 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-ovn
+- as: azure-ipi-private-workers-marketimage-p2-f14
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly-4.10-upgrade-from-stable-4.9.yaml:340]: I think we drop the following `azure-ipi-private-p2-f14` job now, using this one to replace that.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-4.11-upgrade-from-stable-4.10.yaml [hunk 2]
```diff
@@ -337,6 +337,18 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-ovn
+- as: azure-ipi-private-workers-marketimage-p2-f14
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-4.11-upgrade-from-stable-4.10.yaml:340]: I think we drop the following `azure-ipi-private-p2-f14` job now, using this one to replace that.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-stable-4.11.yaml [hunk 3]
```diff
@@ -316,6 +316,18 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-ovn
+- as: azure-ipi-private-workers-marketimage-p2-f7
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-stable-4.11.yaml:319]: I think we drop the following `azure-ipi-private-p2-f14` job now, using this one to replace that.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable-4.12-upgrade-from-stable-4.11.yaml [hunk 4]
```diff
@@ -279,6 +279,18 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-ovn
+- as: azure-ipi-private-workers-marketimage-p2-f14
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable-4.12-upgrade-from-stable-4.11.yaml:282]: I think we drop the following `azure-ipi-private-p2-f14` job now, using this one to replace that.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.12.yaml [hunk 5]
```diff
@@ -369,6 +369,18 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-private
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.12.yaml:371]: By a chance, I found this job is using a wrong workflow, can you help update the workflow to `cucushift-installer-rehearse-azure-ipi-fullyprivate-proxy` together in this PR.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.13.yaml [hunk 6]
```diff
@@ -357,6 +357,18 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-private
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.13.yaml:359]: By a chance, I found this job is using a wrong workflow, can you help update the workflow to cucushift-installer-rehearse-azure-ipi-fullyprivate-proxy together in this PR.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-stable-4.13-upgrade-from-stable-4.12.yaml [hunk 7]
```diff
@@ -362,6 +362,18 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-ovn-ipsec
+- as: azure-ipi-private-workers-marketimage-p2-f7
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-stable-4.13-upgrade-from-stable-4.12.yaml:365]: By a chance, I found the job on line 341 is using a wrong workflow, can you help update the workflow to cucushift-installer-rehearse-azure-ipi-fullyprivate-proxy together in this PR.

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-stable-4.13-upgrade-from-stable-4.12.yaml:365]: If you do not mind, also need to update the same job in `openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.13`, thanks in advance.

**Comment by shellyyang1989** [comment]
Thank you @JianLi-RH! Except for Jianlin's comments, it generally looks good to me.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly-4.10-upgrade-from-stable-4.9.yaml (hunk 1 above)
Comment (JianLi-RH) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly-4.10-upgrade-from-stable-4.9.yaml:340]: @jianlinliu Just to confirm, do you want me to remove following job `azure-ipi-private-p2-f14` in this PR?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-stable-4.13-upgrade-from-stable-4.12.yaml (hunk 7 above)
Comment (JianLi-RH) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-stable-4.13-upgrade-from-stable-4.12.yaml:365]: done

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly-4.10-upgrade-from-stable-4.9.yaml (hunk 1 above)
Comment (shellyyang1989) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly-4.10-upgrade-from-stable-4.9.yaml:340]: @JianLi-RH per my understanding, yes, you're right.

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly-4.10-upgrade-from-stable-4.9.yaml:340]: right

**Comment by JianLi-RH** [comment]
There are too many conflicts, I have to close this MR and open a new one.

**Pull request closed without merging**